apps/notifications/src/panel/templates/block-user.jsx

```diff
@@ -110,8 +108,8 @@ export class UserBlock extends React.Component {
 		}
 
 		if ( home_title ) {
-			var homeClassName =
-				timeIndicator != '' ? 'wpnc__user__meta wpnc__user__bulleted' : 'wpnc__user__meta';
+			const homeClassName =
+				timeIndicator !== '' ? 'wpnc__user__meta wpnc__user__bulleted' : 'wpnc__user__meta';
```

Comment:
`timeIndicator` is defined a few lines above and it is either a React component or an string